widget/entry_test.go

```diff
@@ -315,20 +315,21 @@ func TestEntry_Tapped(t *testing.T) {
 	entry := NewEntry()
 	entry.SetText("MMM")
 
-	test.Tap(entry)
```

Comment:
This test should not be changed.
I am guessing the change was required to work around the fact that you removed the Tapped() handler.
This test must stay in place to ensure we have good cross platform support!